src/components/Button/OFButton.js

```diff
@@ -0,0 +1,29 @@
+import {Button as UtrechtButton} from '@utrecht/component-library-react';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// Temporary until the aria-disabled is set on the Utrecht button
+const OFButton = ({disabled, children, ...extraProps}) => {
+  const {onClick: onClickHandler, ...otherProps} = extraProps;
+
+  if (disabled) otherProps.className = 'utrecht-button--disabled';
+
+  const onClick = event => {
+    if (disabled) event.preventDefault();
+
+    if (onClickHandler) onClickHandler(event);
+  };
+
+  return (
+    <UtrechtButton aria-disabled={disabled} onClick={onClick} {...otherProps}>
```

Comment:
is the `aria-disabled` properly stringified here? It's a bit weird that we're passing a boolean but the rendered HTML should be the string `"true"` or `"false"`.